srcpkgs/nody-greeter/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'nody-greeter'
+pkgname=nody-greeter
+version=1.3.1
+revision=1
+archs="i686 x86_64"
+build_style=meta
+bootstrap=true
```

Comment:
It isn't a bootstrap package. If you see error trying to build it first time, refer to Manual.